### PR TITLE
Update to use new methods in curation_concerns

### PR DESCRIPTION
**Pull request by jcoyne**
@projecthydra/sufia-code-reviewers

### CODE REVIEW

File: spec/search_builder/sufia/single_admin_set_search_builder_spec.rb [hunk 1]
```diff
@@ -7,6 +7,6 @@
   let(:builder) { described_class.new(context) }
   describe "#query" do
     subject { builder.with(id: '123').query.fetch('fq') }
-    it { is_expected.to match_array ["", "_query_:\"{!field f=has_model_ssim}AdminSet\"", "_query_:\"{!field f=id}123\""] }
+    it { is_expected.to match_array ["", "{!terms f=has_model_ssim}AdminSet", "_query_:\"{!field f=id}123\""] }
```
Comment (jeremyf) [inline spec/search_builder/sufia/single_admin_set_search_builder_spec.rb]: This seems like a funny change relative to the above changes.

Comment (jcoyne) [inline spec/search_builder/sufia/single_admin_set_search_builder_spec.rb]: These changes were due to separate upstream changes

**Comment by cbeer** [comment]
Do we need a CC release that we can pin our dependency against?

**Comment by jcoyne** [comment]
@cbeer We talked about this last week.  We have a few outstanding CC PRs to merge prior to a release.

**Comment by cbeer** [comment]
👌

**Pull request merged**